src/lesson-1-8-demo/Multiplier.jsx

```diff
@@ -1,34 +1,20 @@
 import { memo, useCallback, useMemo, useState } from "react";
 
-const DoubleCounter = () => {
+const Multiply = () => {
   const [aIncrement, setAIncrement] = useState(1);
   const [bIncrement, setBIncrement] = useState(1);
   const [countA, setCountA] = useState(0);
   const [countB, setCountB] = useState(0);
 
-  const incrementCountA = () => setCountA(countA + aIncrement);
-  const incrementCountB = () => setCountB(countB + bIncrement);
+  const incrementCountA = useCallback(() => {
+    setCountA((a) => a + aIncrement);
+  }, [aIncrement]);
```

Comment:
`aIncrement` is a dependency of `useCallback`. this means the `useCallback` will only return a new callback handler _if_ `aIncrement` changes